_posts/2017-11-21-Installing-Jenkins-with-PowerShell-DSC.md

```diff
@@ -0,0 +1,745 @@
+---
+layout: post
+title: Installing Jenkins with PowerShell DSC
+author: Rinorragi
+date: 2017-11-21 13:20:00 +0200
+excerpt: How to use PowerShell DSC in installing your continuous integration server.
+categories: 
+- Episerver
+tags: 
+- Episerver 
+- DOTNET 
+- DevOps
+- PowerShell
+- Jenkins
+---
+I have been struggling to find a good reference about how to setup a Jenkins environment in Microsoft environment with automatic installation script. So I decided to write a blog post about it. I also want to write about how to do continuous integration with Episerver DXC but felt that I need to first tell how to setup the Jenkins. Our aim is to put up a Jenkins server that can build .NET FrameWork MVC application and front end. If you are not a reader type of a person, then [here](https://github.com/solita/powershell-dsc-jenkins) is a GitHub link to example scripts.
+
+## About the PowerShell DSC
+
+You might not be familiar with the PowerShell DSC and it comes from Desired Stage Configuration. It has been there now for years still in my experience most .NET developers are unfamiliar with it. DSC exists for various reasons:
```

Comment:
- [ ] Ensimmäinen lause hankala. Olisiko parempi ensimmäiseksi lauseeksi: "The acronym "DSC" comes from Desired Stage Configuration."
- [ ] "...for years. Still, in my experience, most ..."